Cargo.toml

```diff
@@ -59,13 +59,13 @@ sp-version = { default-features = false, features = ["serde"], version = "31.0"
 frame-support = { optional = true, version = "30.0" }
 
 # local deps
-ac-compose-macros = { version = "0.5", default-features = false }
-ac-node-api = { version = "0.6", default-features = false }
-ac-primitives = { version = "0.9", default-features = false }
+ac-compose-macros = { version = "0.5", path = "compose-macros", default-features = false }
```

Comment:
using local paths even if published to crates.io avoids duplicates when patching